playwright/integrations/unauthenticated-chat-with-prechat.spec.ts

```diff
@@ -57,4 +57,142 @@ test.describe('UnauthenticatedChat @UnauthenticatedChatWithPrechat', () => {
         const { preChatResponse: preChatResponseData } = requestPostData;
         expect(optionalParams.preChatResponse).toEqual(preChatResponseData);
     });
+
+    test('ChatSDK.getPreChatSurvey() should return the survey in JSON format', async ({ page }) => {
+
+        await page.goto(testPage);
+
+        const optionalParams = {
```

Comment:
We do not need to call `chatSDK.startChat()` & `chatSDK.endChat()`. We can simply validate the prechat survey after `chatSDK.initialize()`

You simply need to validate the `typeof preChatSurvey === 'object'`

Reply:
Removed and added the validation.